command/operator_debug_test.go

```diff
@@ -176,16 +173,21 @@ func TestDebug_NodeClass(t *testing.T) {
 			args:         []string{"-address", url, "-duration", "250ms", "-server-id", "all", "-node-id", "all", "-node-class", "clientb", "-max-nodes", "2"},
 			expectedCode: 0,
 			expectedOutputs: []string{
-				"Starting debugger",
-				"Created debug archive",
+				"Servers: (1/1)",
+				"Clients: (1/3)",
 				"Node Class: clientb",
+				"Created debug archive",
```

Comment:
The order of these outputs don't seem to reflect the order they're emitted? The test doesn't assert an order either, but do we care about that at all?

Reply:
I rearranged the outputs to match the order they are written to the screen, but that was just for clarity and consistency.  The order really doesn't matter.